src/main/java/com/rackspace/salus/policy/manage/entities/TenantMetadata.java

```diff
@@ -38,7 +40,12 @@
 
 @Entity
 @Table(name = "tenant_metadata", indexes = {
-    @Index(name = "by_tenant", columnList = "tenant_id")
+    @Index(name = "by_tenant", columnList = "tenant_id"),
+    @Index(name = "by_account_type", columnList = "account_type")
+})
+@NamedQueries({
+    @NamedQuery(name = "Tenant.getByAccountType",
```

Comment:
Nit:
```suggestion
    @NamedQuery(name = "TenantMetadata.getByAccountType",
```

BTW, if you use IntelliJ's rename refactoring, it'll change the referencing code.

Reply:
I'm still not entirely sold on my naming choice of `TenantMetadata` which is why I swayed away from it, but I'll stick with it for now.